packages/core/src/interaction-output.ts

```diff
@@ -78,6 +78,14 @@ export class InteractionOutput implements WoT.InteractionOutput {
     }
 
     async value<T extends WoT.DataSchemaValue>(): Promise<T> {
+        // is there any value expected at all?
+        if (this.schema == null) {
+            warn(
+                `No schema defined. Hence undefined is reported for value() function. If you need more control consider using arrayBuffer().`
+            );
```

Comment:
The warning might confuse people that call `value()` on actions that do not have an return value. We could also just use `debug`.
@relu91 ?

Reply:
What about:
```suggestion
            warn(
                `No schema defined. Hence undefined is reported for value() function. If you are invoking an action with no output that is on purpose otherwise prefer using arrayBuffer().`
            );
```

Reply:
Suggestion is fine by me...  I am not sure though whether it should read "prefer using". I think it should still be "consider using". Anyhow, no strong objection 👍

Reply:
Ok for "consider using"

Reply:
Great. Fixed with https://github.com/eclipse-thingweb/node-wot/pull/1230/commits/be93623330bb2be13751f25095721b0d8e8afa2d.